plancton/__init__.py

```diff
@@ -217,11 +217,12 @@ def _read_conf(self):
 
   # Set up monitoring target.
   def _influxdb_setup(self):
-    if not self.conf["influxdb_url"]:
-      self.streamer = lambda *x,**y: True
-      return
-    baseurl,db = self.conf["influxdb_url"].split("#")
-    self.streamer = InfluxDBStreamer(baseurl=baseurl, database=db)
+    self.streamers = []
+    if not isinstance(self.conf["influxdb_url"], list):
```

Comment:
This does not cover the case where `self.conf["influxdb_url"]` is not defined. You should:

```
self.conf.get("influxdb_url", [])
```

instead.